.setup/distro_setup/setup_distro.sh

```diff
@@ -88,15 +88,17 @@ if [ ${VAGRANT} == 1 ]; then
 ############################################################
 ${DISTRO_LINE}
 ##                                                        ##
-##  All user accounts have same password unless otherwise ##
-##  noted below. The following user accounts exist:       ##
-##    vagrant/vagrant, root/vagrant, submitty_dbuser,     ##
-##    submitty_php, submitty_cgi, submitty_daemon, ta,    ##
-##    instructor, developer, postgres                     ##
+##  All user accounts have same password as name.         ##
 ##                                                        ##
-##  The following accounts have database accounts         ##
-##  with same password as above:                          ##
-##    submitty_dbuser, postgres, root, vagrant            ##
+##  The following accounts are system users:              ##
+##    vagrant, root, submitty_php, submitty_cgi,          ##
+##    submitty_daemon, ta, instructor, postgres           ##
+##                                                        ##
+##  The following accounts are database accounts:         ##
+##    submitty_dbuser, root, vagrant                      ##
```

Comment:
postgres user has a DB account (and is the account we use to create the `submitty_dbuser` and `vagrant` ones), root does not.

```
root@vagrant:/usr/local/submitty# su postgres
postgres@vagrant:/usr/local/submitty$ psql
psql (10.8 (Ubuntu 10.8-0ubuntu0.18.04.1))
Type "help" for help.

postgres=# \du
                                      List of roles
    Role name    |                         Attributes                         | Member of
-----------------+------------------------------------------------------------+-----------
 postgres        | Superuser, Create role, Create DB, Replication, Bypass RLS | {}
 submitty_dbuser | Superuser, Create role, Create DB                          | {}
 vagrant         | Superuser, Create role, Create DB                          | {}
```